infra/upload_images/container_ocr/handler/tests/test_overlay.py

```
@@ -0,0 +1,997 @@
+        ocr_job = SimpleNamespace(
+            image_id=_IMG_ID, receipt_id=1,
+            reocr_region=region, s3_bucket="b", s3_key="k",
+        )
+        routing = SimpleNamespace(
+            s3_bucket="b", s3_key="r.json",
+            status=None, receipt_count=None, updated_at=None,
+        )
+
+        proc.dynamo.list_receipt_words_from_receipt.return_value = existing_words
+        proc.dynamo.list_receipt_word_labels_for_receipt.return_value = (
+            [SimpleNamespace(line_id=l.line_id, word_id=l.word_id) for l in (labels or [])],
```

Ruff (0.15.1)

[error] Ambiguous variable name: `l`

(E741)